### PR TITLE
Use port from config and default port parenthesis bug fix

### PR DESCRIPTION
**Pull request by jdkanani**

### CODE REVIEW

**Comment by jdkanani** [comment]
Sorry, forgot to fix in last PR.

File: src/config/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const debug = env !== 'production'
 dotenv.config({ path: configFile, debug: debug })
 
 // retrieve port from env or use default
-const port = process.env.PORT || env === 'production' ? 5000 : 5002
+const port = process.env.PORT || (env === 'production' ? 5000 : 5002)
```
Comment (jdkanani) [inline src/config/index.ts:19]: It was taking `(process.env.PORT || env === 'production') ? 5000 : 5002` before

Comment (pedrouid) [reply src/config/index.ts:19]: Good catch! 👍

Comment (jdkanani) [reply src/config/index.ts:19]: 👍

**Pull request merged**